File: edit-extension/gptedit/src/fizzbuzz.ts
const fizzbuzz = () => {
  for (let i = 1; i <= 100; i++) {
    calculateFizzBuzzOutput(i);
  }
};
function calculateFizzBuzzOutput(i: number) {
  let output = "";
  if (i % 3 === 0) output += "Fizz";
  if (i % 5 === 0) output += "Buzz";
  const result = output || i;
  console.log(result);
}

File: fizzbuzz-with-typist/fizzbuzz.py
def fizzbuzz(n, components=None):
    if components is None:
        components = [
            ("Fizz", 3),
            ("Buzz", 5),
            ("Whizz", 7),
            ("Bang", 11),
            ("Bong", 13),
        ]

    return "".join(
        [word for word, divisor in components if n % divisor == 0]) or str(n)
